src/components/LayersDrawer/LayersDrawer.tsx

```
@@ -58,39 +56,43 @@ export default function LayersDrawer({ mapLayers, setMapLayers, selectedYear }:
     [activeRasterLayerId, setMapLayers],
   )
 
+  const toggleSubLayer = useCallback(
+    (event: React.ChangeEvent<HTMLInputElement>, checked: boolean) => {
```

ESLint

[error] 'checked' is defined but never used.

(@typescript-eslint/no-unused-vars)